R/queue_base.R

```diff
@@ -250,7 +250,9 @@ queue_base <- R6::R6Class(
     ##'
     ##' @param name Optional name for a created bundle
     ##'
-    ##' @param depends_on Optional vector of task ids to depend on
+    ##' @param depends_on Optional task ids to depend on.
```

Comment:
I believe that roxygen2 can do `@inheritParams` from a different package so you can do `@inheritParams context::bulk_task_save` here and delete this specific docs and then you'd not need to duplicate the text? Not 100%, give it a try.

Reply:
Looks like that doesn't work inside R6 classes which support a more [limited set of tags](https://roxygen2.r-lib.org/articles/rd-other.html#r6). I've updated to link to the `context` docs though.